h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -297,7 +340,7 @@ static int getMaxConnectionsPerNode(final int cloudSize, final short nThreads, f
      * @param maxTotalConnections Maximal number of total connections to be opened by the whole cluster
      * @return Number of connections to open per node, within given minmal and maximal range
      */
-    private static final int calculateLocalConnectionCount(final int maxTotalConnections, final int cloudSize,
+    private static int calculateLocalConnectionCount(final int maxTotalConnections, final int cloudSize,
```

Comment:
Why not non-final ? Not that it matters ... Just a question, nothing really important in our API design I believe.